src/compas/files/gltf/gltf_content.py

```diff
@@ -403,30 +490,132 @@ def visit(node, key):
 # Main
 # ==============================================================================
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
I see the value of the changes you've made to this sections pointing out caveats to adding images and textures.  But I also see the value of providing a bare minimum example of how to create a GLTF.  Do you think it would be worthwhile to keep both?  

The other changes in the PR look great!

Reply:
thank you beverly! 
yes, for now, I would like to keep both, as there might be further improvements needed.